core/include/msg.h

```diff
@@ -362,9 +362,9 @@ int msg_reply_int(msg_t *m, msg_t *reply);
  * @brief Check how many messages are available in the message queue
  *
  * @return Number of messages available in our queue on success
- * @return -1, if no caller's message queue is initialized
+ * @return 0, if no caller's message queue is initialized
```

Comment:
```suggestion
 * @retval 0    if no caller's message queue is initialized or the queue is full
```

Reply:
no, it returns 0 if the queue is either empty or there's no queue. if it is full, it returns the number of available messages.